app/forms/sufia/forms/work_form.rb

```diff
@@ -7,5 +7,17 @@ def rendered_terms
                :visibility_after_embargo, :visibility_during_lease,
                :lease_expiration_date, :visibility_after_lease, :visibility, :thumbnail_id, :representative_id, :ordered_member_ids]
     end
+
+    # override Curation Concerns Work Form, Sufia allows multiple values for rights
+    def self.multiple?(term)
+      case term.to_s
+      when 'rights'
+        true
+      when 'ordered_member_ids'
```

Comment:
How does this clause solve the issue you reported?  It doesn't seem releated to me. If it's not related, it shouldn't go in this PR.